src/main/java/core/basesyntax/Print.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Print {
+    void print();
```

Comment:
I think figure should `draw` instead of `print`. In the task: `To 'draw' means to print`.
Lets rename it to `Drawable` with `void draw()`.